src/types/onyx/Card.ts

```diff
@@ -8,18 +8,22 @@ type Card = {
     bank: string;
     availableSpend: number;
     domainName: string;
-    maskedPan?: string; // do not reference, removing as part of Expensify/App#27943
     lastFourPAN?: string;
-    cardName: string;
-    cardTitle: string; // used only for virtual limit cards
-    limitType: ValueOf<typeof CONST.EXPENSIFY_CARD.LIMIT_TYPES>;
-    isAdminIssuedVirtualCard: boolean;
     isVirtual: boolean;
```

Comment:
Commented in the BE PR but wondering if we should deprecate this and use nameValuePairs.isVirtual instead